File: Loadshop.DomainServices/Loadshop/Services/Data/LoadAuditLogData.cs
using System;

namespace Loadshop.DomainServices.Loadshop.Services.Data
{
    public class LoadAuditLogData
    {
        public long LoadAuditLogId { get; set; }
        public string AuditTypeId { get; set; }
        public Guid LoadId { get; set; }
        public string ReferenceLoadId { get; set; }
        public Guid CustomerId { get; set; }
        public short Stops { get; set; }
        public DateTime PickupDtTm { get; set; }
        public DateTime DeliveryDtTm { get; set; }
        public int Miles { get; set; }
        public decimal LineHaulRate { get; set; }
        public decimal FuelRate { get; set; }
        public int Weight { get; set; }
        public int Cube { get; set; }
        public string Commodity { get; set; }
        public string EquipmentId { get; set; }
        public bool IsHazMat { get; set; }
        public string Comments { get; set; }
        public Guid UserId { get; set; }
        public string Scac { get; set; }
        public decimal? ContractRate { get; set; }
        public string CarrierName { get; set; }
        public string Username { get; set; }
        public string FirstName { get; set; }
        public string Email { get; set; }
        public string Phone { get; set; }
        public DateTime CreateDtTm { get; set; }
    }
}

File: Loadshop.Web.UI/src/app/shared/models/load-audit-log-data.ts
// eslint-disable-next-line @typescript-eslint/interface-name-prefix
export interface ILoadAuditLogData {
  loadAuditLogId: number;
  auditTypeId: string;
  loadId: string;
  referenceLoadId: string;
  customerId: string;
  stops: number;
  pickupDtTm: Date;
  deliveryDtTm: Date;
  miles: number;
  lineHaulRate: number;
  fuelRate: number;
  weight: number;
  cube: number;
  commodity: string;
  equipmentId: string;
  isHazMat: boolean;
  comments: string;
  userId: string;
  scac: string;
  contractRate: number;
  carrierName: string;
  userName: string;
  firstName: string;
  email: string;
  phone: string;
}

export class LoadAuditLogData implements ILoadAuditLogData {
  constructor(init?: Partial<LoadAuditLogData>) {
    Object.assign(this, init);
  }

  loadAuditLogId: number;
  auditTypeId: string;
  loadId: string;
  referenceLoadId: string;
  customerId: string;
  stops: number;
  pickupDtTm: Date;
  deliveryDtTm: Date;
  miles: number;
  lineHaulRate: number;
  fuelRate: number;
  weight: number;
  cube: number;
  commodity: string;
  equipmentId: string;
  isHazMat: boolean;
  comments: string;
  userId: string;
  scac: string;
  contractRate: number;
  carrierName: string;
  userName: string;
  firstName: string;
  email: string;
  phone: string;
  createDtTm: Date;
}
